packages/block-editor/src/components/off-canvas-editor/index.js

```diff
@@ -227,6 +232,7 @@ function OffCanvasEditor(
 				>
 					<ListViewContext.Provider value={ contextValue }>
 						<ListViewBranch
+							parentId={ parentId }
```

Comment:
is a better name for this rootBlockId or something as the list view is not really a child of a block?

Reply:
The component ListViewBranch already had the parentId property, I'm just passing the property in a case where we were not passing it before, but the prop already existed.
The list view component also calls it parentId https://github.com/WordPress/gutenberg/blob/a0622539f50011643ebd378d1130e491558e8c95/packages/block-editor/src/components/list-view/branch.js#L93.
I can change it to rootBlockId on the offsite canvas editor and the list view if you prefer. I don't have a specific preference.

Reply:
Oh I lost this fact in reviewing the code, thought we just added `parentId`. Let's not change anything then :)